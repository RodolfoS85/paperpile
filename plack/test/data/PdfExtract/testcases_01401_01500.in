---
file: ../../../PDFREPOSITORY/001401.pdf
---
file: ../../../PDFREPOSITORY/001402.pdf
---
file: ../../../PDFREPOSITORY/001403.pdf
---
file: ../../../PDFREPOSITORY/001404.pdf
---
file: ../../../PDFREPOSITORY/001405.pdf
---
file: ../../../PDFREPOSITORY/001406.pdf
---
file: ../../../PDFREPOSITORY/001407.pdf
---
file: ../../../PDFREPOSITORY/001408.pdf
---
file: ../../../PDFREPOSITORY/001409.pdf
---
file: ../../../PDFREPOSITORY/001410.pdf
---
file: ../../../PDFREPOSITORY/001411.pdf
---
file: ../../../PDFREPOSITORY/001412.pdf
---
file: ../../../PDFREPOSITORY/001413.pdf
---
file: ../../../PDFREPOSITORY/001414.pdf
---
file: ../../../PDFREPOSITORY/001415.pdf
---
file: ../../../PDFREPOSITORY/001416.pdf
---
file: ../../../PDFREPOSITORY/001417.pdf
---
file: ../../../PDFREPOSITORY/001418.pdf
---
file: ../../../PDFREPOSITORY/001419.pdf
---
file: ../../../PDFREPOSITORY/001420.pdf
---
file: ../../../PDFREPOSITORY/001421.pdf
---
file: ../../../PDFREPOSITORY/001422.pdf
---
file: ../../../PDFREPOSITORY/001423.pdf
---
file: ../../../PDFREPOSITORY/001424.pdf
---
file: ../../../PDFREPOSITORY/001425.pdf
---
file: ../../../PDFREPOSITORY/001426.pdf
---
file: ../../../PDFREPOSITORY/001427.pdf
---
file: ../../../PDFREPOSITORY/001428.pdf
---
file: ../../../PDFREPOSITORY/001429.pdf
---
file: ../../../PDFREPOSITORY/001430.pdf
---
file: ../../../PDFREPOSITORY/001431.pdf
---
file: ../../../PDFREPOSITORY/001432.pdf
---
file: ../../../PDFREPOSITORY/001433.pdf
---
file: ../../../PDFREPOSITORY/001434.pdf
---
file: ../../../PDFREPOSITORY/001435.pdf
---
file: ../../../PDFREPOSITORY/001436.pdf
---
file: ../../../PDFREPOSITORY/001437.pdf
---
file: ../../../PDFREPOSITORY/001438.pdf
---
file: ../../../PDFREPOSITORY/001439.pdf
---
file: ../../../PDFREPOSITORY/001440.pdf
---
file: ../../../PDFREPOSITORY/001441.pdf
---
file: ../../../PDFREPOSITORY/001442.pdf
---
file: ../../../PDFREPOSITORY/001443.pdf
---
file: ../../../PDFREPOSITORY/001444.pdf
---
file: ../../../PDFREPOSITORY/001445.pdf
---
file: ../../../PDFREPOSITORY/001446.pdf
---
file: ../../../PDFREPOSITORY/001447.pdf
---
file: ../../../PDFREPOSITORY/001448.pdf
---
file: ../../../PDFREPOSITORY/001449.pdf
---
file: ../../../PDFREPOSITORY/001450.pdf
---
file: ../../../PDFREPOSITORY/001451.pdf
---
file: ../../../PDFREPOSITORY/001452.pdf
---
file: ../../../PDFREPOSITORY/001453.pdf
---
file: ../../../PDFREPOSITORY/001454.pdf
---
file: ../../../PDFREPOSITORY/001455.pdf
---
file: ../../../PDFREPOSITORY/001456.pdf
---
file: ../../../PDFREPOSITORY/001457.pdf
---
file: ../../../PDFREPOSITORY/001458.pdf
---
file: ../../../PDFREPOSITORY/001459.pdf
---
file: ../../../PDFREPOSITORY/001460.pdf
---
file: ../../../PDFREPOSITORY/001461.pdf
---
file: ../../../PDFREPOSITORY/001462.pdf
---
file: ../../../PDFREPOSITORY/001463.pdf
---
file: ../../../PDFREPOSITORY/001464.pdf
---
file: ../../../PDFREPOSITORY/001465.pdf
---
file: ../../../PDFREPOSITORY/001466.pdf
---
file: ../../../PDFREPOSITORY/001467.pdf
---
file: ../../../PDFREPOSITORY/001468.pdf
---
file: ../../../PDFREPOSITORY/001469.pdf
---
file: ../../../PDFREPOSITORY/001470.pdf
---
file: ../../../PDFREPOSITORY/001471.pdf
---
file: ../../../PDFREPOSITORY/001472.pdf
---
file: ../../../PDFREPOSITORY/001473.pdf
---
file: ../../../PDFREPOSITORY/001474.pdf
---
file: ../../../PDFREPOSITORY/001475.pdf
---
file: ../../../PDFREPOSITORY/001476.pdf
---
file: ../../../PDFREPOSITORY/001477.pdf
---
file: ../../../PDFREPOSITORY/001478.pdf
---
file: ../../../PDFREPOSITORY/001479.pdf
---
file: ../../../PDFREPOSITORY/001480.pdf
---
file: ../../../PDFREPOSITORY/001481.pdf
---
file: ../../../PDFREPOSITORY/001482.pdf
---
file: ../../../PDFREPOSITORY/001483.pdf
---
file: ../../../PDFREPOSITORY/001484.pdf
---
file: ../../../PDFREPOSITORY/001485.pdf
---
file: ../../../PDFREPOSITORY/001486.pdf
---
file: ../../../PDFREPOSITORY/001487.pdf
---
file: ../../../PDFREPOSITORY/001488.pdf
---
file: ../../../PDFREPOSITORY/001489.pdf
---
file: ../../../PDFREPOSITORY/001490.pdf
---
file: ../../../PDFREPOSITORY/001491.pdf
---
file: ../../../PDFREPOSITORY/001492.pdf
---
file: ../../../PDFREPOSITORY/001493.pdf
---
file: ../../../PDFREPOSITORY/001494.pdf
---
file: ../../../PDFREPOSITORY/001495.pdf
---
file: ../../../PDFREPOSITORY/001496.pdf
---
file: ../../../PDFREPOSITORY/001497.pdf
---
file: ../../../PDFREPOSITORY/001498.pdf
---
file: ../../../PDFREPOSITORY/001499.pdf
---
file: ../../../PDFREPOSITORY/001500.pdf
