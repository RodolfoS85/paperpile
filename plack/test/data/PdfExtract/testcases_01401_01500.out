---
authors: 'Voß, Björn and Georg, Jens and Schön, Verena and Ude, Susanne and Hess, Wolfgang R'
doi: 10.1186/1471-2164-10-123
title: Biocomputational prediction of non-coding RNAs in model cyanobacteria
---
authors: 'Voß, Björn and Giegerich, Robert and Rehmsmeier, Marc'
doi: 10.1186/1741-7007-4-5
title: Complete probabilistic analysis of RNA shapes
---
authors: 'Voß, Björn and Gierga, Gregor and Axmann, Ilka M and Hess, Wolfgang R'
doi: 10.1186/1471-2164-8-375
title: A motif-based search in bacterial genomes identifies the ortholog of the small RNA Yfr1 in all lineages of cyanobacteria
---
authors: 'Vuyisich, Momchilo and Beal, Peter A.'
title: Controlling Protein Activity with Ligand-Regulated RNA Aptamers
---
authors: 'Wachtel, Chaim and Li, Binghui and Sperling, Joseph and Sperling, Ruth'
doi: 10.1261/rna.7480804
title: Stop codon-mediated suppression of splicing is a novel nuclear scanning mechanism not affected by elements of protein synthesis and NMD
---
authors: 'Wagner, Andreas'
title: 'Neutralism and selectionism: a network-based reconciliation'
---
authors: 'Wagner, Gunte P. and Amemiya, Chris and Ruddle, Frank'
doi: 10.1073/pnas.2536656100
title: Hox cluster duplications and the opportunity for evolutionary novelties
---
authors: 'Wagner, Andreas'
title: A computational genomics approach to the identification of gene networks
---
authors: 'Wagner, E Gerhart H'
doi: 10.1210/me.2008-0386
title: 'Kill the messenger: bacterial antisense RNA promotes mRNA decay'
---
authors: 'Wakefield, Matthew J and Maxwell, Peter and Huttley, Gavin A'
doi: 10.1186/1471-2105-6-130
title: 'Vestige: Maximum likelihood phylogenetic footprinting'
---
authors: 'Waldisphl, Jrme and ODonnell, Charles W. and Will, Sebastian and Devadas, Srinivas and Backofen, Rolf and Berger, Bonnie'
title: Simultaneous Alignment and Folding of Protein Sequences
---
authors: 'Waldispuhl, Jerome and Devadas, Srinivas and Berger, Bonnie and Clote, Peter'
doi: 10.1371/journal.pcbi.1000124
title: Efficient Algorithms for Probing the RNA Mutation Landscape
---
authors: 'Wall, P Kerr and Leebens-Mack, Jim and Chanderbali, André S and Barakat, Abdelali and Wolcott, Erik and Liang, Haiying and Landherr, Lena and Tomsho, Lynn P and Hu, Yi and Carlson, John E and Ma, Hong and Schuster, Stephan C and Soltis, Douglas E and Soltis, Pamela S and Altman, Naomi and dePamphilis, Claude W'
doi: 10.1186/1471-2164-10-347
title: Comparison of next generation sequencing technologies for transcriptome characterization
---
authors: 'Wallace, Iain M. and OSullivan, Orla and Higgins, Desmond G.'
title: Evaluation of Iterative Alignment Algorithms for Multiple Alignment
---
authors: 'Walsh, Toby'
title: Stochastic Constraint Programming
---
authors: 'Wang, Zhuozhi and Zhang, Kaizhong'
title: Alignment between Two RNA Structures
---
authors: 'Wang, Lusheng and Ma, Bin and Li, Ming'
title: Fixed topology alignment with recombination
---
authors: 'Wang, Bing-Bing and Brendel, Volker'
doi: 10.1073/pnas.0602039103
title: Genomewide comparative analysis of alternative splicing in plants
---
authors: 'Wang, Jun and Chang, Yao-Fu and Hamilton, John I. and Wilkinson, Miles F.'
doi: 10.1016/S1097276502006354
title: 'Nonsense-Associated Altered Splicing: A Frame-Dependent Response Distinct from Nonsense-Mediated Decay'
---
authors: 'Wang, Lei and Duke, Lindsay and Zhang, Peter S. and Arlinghaus, Ralph B. and Symmans, W. Fraser and Sahin, Aysegul and Mendez, Richard and Le Dai, Jia'
title: Alternative Splicing Disrupts a Nuclear Localization Signal in Spleen Tyrosine Kinase That Is Required for Invasion Suppression in Breast Cancer
---
authors: 'Wang, Tairan'
title: Symmetry and designability for lattice protein models
---
authors: 'Wang, Shu and Gutell, Robin R. and Miranker, Daniel P.'
title: Biclustering As A Method For RNA Local Multiple Sequence Alignment
---
authors: 'Wang, Zhining and Shuen Lo, H. and Yang, Howard and Gere, Sheryl and Hu, Ying and Buetow, Kenneth H. and Lee, Maxwell P.'
title: Computational Analysis and Experimental Validation of Tumor-associated Alternative RNA Splicing in Human Cancer
---
authors: 'Wang, Xiu-Jie and Reyes, José L and Chua, Nam-Hai and Gaasterland, Terry'
title: Prediction and identification of Arabidopsis thaliana microRNAs and their mRNA targets
---
authors: 'Wang, Haidong and Segal, Eran and Ben-Hur, Asa and Li, Qian-Ru and Vidal, Marc and Koller, Daphne'
doi: 10.1186/gb-2007-8-9-r192
title: 'InSite: a computational method for identifying protein-protein interaction binding sites on a proteome-wide scale'
---
authors: 'Wang, Victoria E. H. and Tantin, Dean and Chen, Jianzhu and Sharp, Phillip A.'
doi: 10.1073/pnas.0307304101
title: B cell development and immunoglobulin transcription in Oct-1-deficient mice
---
authors: 'Wang, Peng and Yan, Bo and Guo, Jun-tao and Hicks, Chindo and Xu, Ying'
doi: 10.1073/pnas.0506770102
title: Structural genomics analysis of alternative splicing and application to isoform structure modeling
---
authors: 'Washietl, Stefan and Hofacker, Ivo L.'
title: Consensus folding of aligned sequences as a new measure for the detection of functional RNAs by comparative genomics
---
authors: 'Washietl, Stefan and Hofacker, Ivo L and Lukasser, Melanie and Hüttenhofer, Alexander and Stadler, Peter F'
doi: 10.1038/nbt1144
title: Mapping of conserved RNA secondary structures predicts thousands of functional noncoding RNAs in the human genome
---
authors: 'Washietl, Stefan and Hofacker, Ivo L. and Stadler, Peter F.'
doi: 10.1073/pnas.0409169102
title: Fast and reliable prediction of noncoding RNAs
---
authors: 'Wassarman, Karen M. and Kiley, Patricia J.'
doi: 10.1128/JB.01316-09
title: Global Approaches for Finding Small RNA and Small Open Reading Frame Functions
---
authors: 'Wassarman, Karen M. and Repoila, Francis and Rosenow, Carsten and Storz, Gisela and Gottesman, Susan'
doi: 10.1101/gad.901001
title: Identification of novel small RNAs using comparative genomics and microarrays
---
authors: 'Wassarman, Karen M'
doi: 10.1016/j.mib.2007.03.008
title: '6S RNA: a small RNA regulator of transcription'
---
authors: 'Wasserman, Wyeth W. and Palumbo, Michael and Thompson, William and Fickett, James W. and Lawrence, Charles E.'
title: Human-mouse genome comparisons to locate regulatory sites
---
authors: 'Wasserman, Wyeth W. and Sandelin, Albin'
doi: 10.1038/nrg1315
title: APPLIED BIOINFORMATICS FOR THE IDENTIFICATION OF REGULATORY ELEMENTS
---
authors: '{Mouse Genome Sequencing Consortium}'
title: Initial sequencing and comparative analysis of the mouse genome
---
authors: 'Waugh, Allison and Gendron, Patrick and Altman, Russ and Brown, James W. and Case, David and Gautheret, Daniel and Harvey, Stephen C. and Leontis, Neocles and Westbrook, John and Westhof, Eric and Zuker, Michael and Major, FRANÇOIS'
title: 'RNAML: A standard syntax for exchanging RNA information'
---
authors: 'Weaver, Keith E'
doi: 10.1016/j.mib.2007.03.002
title: Emerging plasmid-encoded antisense RNA regulated systems
---
authors: 'Weber, Hansjörg and Brecker, Lothar'
title: Online NMR for monitoring biocatalysed reactions
---
authors: 'Weber, James L. and Myers, Eugene W.'
title: Human Whole-Genome Shotgun Sequencing
---
authors: 'Werner, Thomas'
title: Models for prediction and recognition of eukaryotic promoters
---
authors: 'Wernicke, Sebastian'
title: Efficient Detection of Network Motifs
---
authors: 'Wessler, Susan R and Bureau, Thomas E and White, Shawn E'
title: 'LTR-retrotransposons and MITEs: important players in the evolution of plant genomes'
---
authors: 'West, Adam G. and Sharrocks, Andrew D.'
title: MADS-box Transcription Factors Adopt Alternative Mechanisms for Bending DNA
---
authors: 'West, Adam G. and Shore, Paul and Sharrocks, Andrew D.'
title: 'DNA Binding by MADS-Box Transcription Factors: a Molecular Mechanism for Differential DNA Bending'
---
authors: 'Wexler, Ydo'
title: A Study of Accessible Motifs and RNA Folding Complexity
---
authors: 'White, Erica K. and MOORE-JARRETT, Tracy and EARL RULEY, H.'
title: 'PUM2, a novel murine Puf protein, and its consensus RNA-binding site'
---
authors: 'Wilhelm, Thomas'
doi: 10.1103/PhysRevE.76.011911
title: Analysis of structures causing instabilities
---
authors: 'Wilhelm, Thomas and Nasheuer, Heinz-Peter and Huang, Sui'
doi: 10.1074/mcp.M300005-MCP200
title: Physical and Functional Modularity of the Protein Network in Yeast
---
authors: 'Will, Sebastian and Mann, Martin and Backofen, Rolf'
title: Counting Protein Structures by Dynamic Decomposition during Search
---
authors: 'Will, Sebastian and Mann, Martin'
title: Counting Protein Structures by DFS with Dynamic Decomposition
---
arxivid: physics/0201054
authors: 'Hansmann, Ulrich H. E.'
title: Global Optimization by Energy Landscape Paving
---
authors: 'Wilm, Andreas and Mainz, Indra and Steger, Gerhard'
doi: 10.1186/1748-7188-1-19
title: An enhanced RNA alignment benchmark for sequence alignment programs
---
authors: 'WILSON-RAWLS, Jeanne and Molkentin, Jeffery D. and Black, Brian L. and Olson, Eric N.'
title: Activated Notch Inhibits Myogenic Activity of the MADS-Box Transcription Factor Myocyte Enhancer Factor 2C
---
authors: 'Wilson, David S. and Keefe, Anthony D. and Szostak, Jack W.'
doi: 10.1073/pnas.061028198
title: The use of mRNA display to select high-affinity protein-binding peptides
---
authors: 'Wilson, Megan and Koopman, Peter'
title: 'Matching SOX: partner proteins and co-factors of the SOX family of transcriptional regulators'
---
authors: 'Wilson, Charles and Nix, Jay and Szostak, Jack'
title: Functional Requirements for Specific Ligand Recognition by a Biotin-Binding RNA Pseudoknot
---
authors: 'Wilson, J. W. and Ott, C. M. and Honer zu Bentrup, K. and Ramamurthy, R. and Quick, L. and Porwollik, S. and Cheng, P. and McClelland, M. and Tsaprailis, G. and Radabaugh, T. and Hunt, A. and Fernandez, D. and Richter, E. and Shah, M. and Kilcoyne, M. and Joshi, L. and Nelman-Gonzalez, M. and Hing, S. and Parra, M. and Dumars, P. and Norwood, K. and Bober, R. and Devich, J. and Ruggles, A. and Goulart, C. and Rupert, M. and Stodieck, L. and Stafford, P. and Catella, L. and Schurr, M. J. and Buchanan, K. and Morici, L. and McCracken, J. and Allen, P. and Baker-Coleman, C. and Hammond, T. and Vogel, J. and Nelson, R. and Pierson, D. L. and Stefanyshyn-Piper, H. M. and Nickerson, C. A.'
doi: 10.1073/pnas.0707155104
title: Space flight alters bacterial gene expression and virulence and reveals a role for global regulator Hfq
---
authors: 'Wilting, R. and Schorling, S. and Persson, B. C. and Bock, A'
title: 'Selenoprotein Synthesis in Archaea: Identification of an mRNA Element of Methanococcus jannaschii Probably Directing Selenocysteine Insertion'
---
authors: 'Wilusz, Carol J and Wilusz, Jeffrey'
doi: 10.1038/nsmb1037
title: 'Eukaryotic Lsm proteins: lessons from bacteria'
---
authors: 'Wingender, E. and Chen, X. and Hehl, R. and Karas, H. and Liebich, I. and Matys, V. and Meinhardt, T. and Prüß, M. and Reuter, I. and Schacherer, F.'
title: 'TRANSFAC: an integrated system for gene expression regulation'
---
authors: 'Wingender, E. and Chen, X. and Fricke, E. and Geffers, R. and Hehl, R. and Liebich, I. and Krull, M. and Matys, V. and Michael, H. and Ohnhäuser, R. and Prüß, M. and Schacherer, F. and Thiele, S. and Urbach, S.'
title: The TRANSFAC system on gene expression regulation
---
authors: 'Wingender, E. and Dietze, P. and Karas, H. and Knüppel, R.'
title: 'TRANSFAC: a database on transcription factors and their DNA binding sites'
---
authors: 'Winkler, Wade C. and Nahvi, Ali and Roth, Adam and Collins, Jennifer A. and Breaker, Ronald R.'
title: Control of gene expression by a natural metabolite-responsive ribozyme
---
authors: 'Winter, Jennifer and Lehmann, Tanja and Krauß, Sybille and Trockenbacher, Alexander and Kijas, Zofia and Foerster, John and Suckow, Vanessa and Yaspo, Marie-Laure and Kulozik, Andreas and Kalscheuer, Vera and Schneider, Rainer and Schweiger, Susann'
doi: 10.1007/s00439-004-1114-x
title: Regulation of the MID1 protein function is fine-tuned by a complex pattern of alternative splicing
---
authors: 'Winter, Kai-Uwe and Weiser, Christof and Kaufmann, Kerstin and Bohne, Arend and Kirchner, Charlotte and Kanno, Akira and Saedler, Heinz and Theißen, Gunter'
title: 'Evolution of Class B Floral Homeotic Proteins: Obligate Heterodimerization Originated from Homodimerization'
---
authors: 'Wolberger, Cynthia'
title: MULTIPROTEIN-DNA COMPLEXES IN TRANSCRIPTIONAL REGULATION
---
authors: 'Wolfinger, Michael T. and Andreas Svrcek-Seiler, W. and Flamm, Christoph and Hofacker, Ivo L. and Stadler, Peter F.'
title: Exact Folding Dynamics of RNA Secondary Structures
---
authors: 'Wolfsberg, Tyra G. and Landsman, David'
title: A comparison of expressed sequence tags (ESTs) to human genomic sequences
---
authors: 'Wolin, Sandra L. and Matera, A. Gregory'
doi: 10.1101/gad.13.1.1
title: The trials and travels of tRNA
---
authors: 'Wollerton, Matthew C. and Gooding, Clare and Wagner, Eric J. and Garcia-Blanco, Mariano A. and Smith, Christopher W. J.'
title: Autoregulation of Polypyrimidine Tract Binding Protein by Alternative Splicing Leading to Nonsense-Mediated Decay
---
authors: 'Wolner, Branden S. and Gralla, Jay D.'
doi: 10.1074/jbc.M008273200
title: TATA-flanking Sequences Influence the Rate and Stability of TATA-binding Protein and TFIIB Binding
---
authors: 'Woodson, S. A.'
title: Recent insights on RNA folding mechanisms from catalytic RNA
---
authors: 'Workman, J. L.'
title: ALTERATION OF NUCLEOSOME STRUCTURE AS A MECHANISM OF TRANSCRIPTIONAL REGULATION
---
authors: 'Wosik, Ewa'
title: Probabilistic Boolean and Bayesian Networks
---
authors: 'Wroe, Richard and Bornberg-Bauer, Erich and Chan, Hue Sun'
doi: 10.1529/biophysj.104.050369
title: 'Comparing Folding Codes in Simple Heteropolymer Models of Protein Evolutionary Landscape: Robustness of the Superfunnel Paradigm'
---
authors: 'Wroe, Richard and Chan, Hue Sun and Bornberg-Bauer, Erich'
doi: '10.2976/1.2739116]'
title: A structural model of latent evolutionary potentials underlying neutral networks in proteins
---
authors: 'Wu, Tzong-Yuan and Hsieh, Chi-Chun and Hong, Jun-Jie and Chen, Chung-Yung and Tsai, Yuh-Show'
doi: 10.1186/1471-2105-10-160
title: 'IRSS: a web-based tool for automatic layout and analysis of IRES secondary structure prediction and searching system in silico'
---
authors: 'Wu, Thomas D. and Nevill-Manning, Craig G. and Brutlag, Douglas L.'
title: Fast probabilistic analysis of sequence function using scoring matrices
---
authors: 'Wuyts, Jan and Van de Peer, Yves and Winkelmans, Tina and De Wachter, Rupert'
title: The European database on small subunit ribosomal RNA
---
authors: 'Wyckoff, Gerald J. and Wang, Wen and Wu, Chung-I'
title: Rapid evolution of male reproductive genes in the descent of man
---
authors: 'Wyrick, John J and Young, Richard A'
title: Deciphering gene expression regulatory networks
---
authors: 'Xayaphoummine, A. and Bucher, T. and Thalmann, F. and Isambert, H.'
doi: 10.1073/pnas.2536430100
title: Prediction and statistics of pseudoknots in RNA structures using exactly clustered stochastic simulations
---
authors: 'Xia, Yu and Huang, Enoch S. and Levitt, Michael and Samudrala, Ram'
doi: 10.1006/jmbi.2000.3835
title: Ab Initio Construction of Protein Tertiary Structures Using a Hierarchical Approach
---
authors: 'Xie, Hanqing and Wasserman, Alon and Levine, Zurit and Novik, Amit and Grebinskiy, Vladimir and Shoshan, Avi and Mintz, Liat'
doi: 10.1101/gr.86902
title: Large-Scale Protein Annotation through Gene Ontology
---
authors: 'Xie, Hanqing and Zhu, Wei-yong and Wasserman, Alon and Grebinskiy, Vladimir and Olson, Andrew and Mintz, Liat'
doi: 10.1006/geno.2002.6841
title: Computational Analysis of Alternative Splicing Using EST Tissue Information
---
authors: 'Xing, Yi and Lee, Christopher'
doi: 10.1073/pnas.0501213102
title: Evidence of functional selection pressure for alternative splicing events that accelerate evolution of protein subsequences
---
authors: 'Xing, Yi and Lee, Christopher J.'
doi: 10.1371/journal.pgen.0010034
title: Protein Modularity of Alternatively Spliced Exons Is Associated with Tissue-Specific Regulation of Alternative Splicing
---
authors: 'Xing, Yi and Wang, Qi and Lee, Christopher'
doi: 10.1534/genetics.106.057919
title: 'Evolutionary Divergence of Exon Flanks: A Dissection of Mutability and Selection'
---
authors: 'Xu, Jinbo and Berger, Bonnie'
title: Fast and Accurate Algorithms for Protein Side-Chain Packing
---
authors: 'Xu, Xing and Ji, Yongmei and Stormo, Gary D.'
doi: 10.1371/journal.pcbi.1000338
title: Discovering cis-Regulatory RNAs in Shewanella Genomes by Support Vector Machines
---
authors: 'Xu, Jinbo and Li, Ming'
title: 'RAPTOR: OPTIMAL PROTEIN THREADING BY LINEAR PROGRAMMING'
---
authors: 'Xu, Ying and Olman, Victor and Xu, Dong'
title: 'Clustering gene expression data using a graph-theoretic approach: an application of minimum spanning trees'
---
authors: 'Xu, Pin-Xian and Zheng, Weiming and Huang, Li and Maire, Pascal and Laclef, Christine and Silvius, Derek'
doi: 10.1242/dev.00536
title: Six1 is required for the early organogenesis of mammalian kidney
---
authors: 'Yada, Tetsushi and Totoki, Yasushi and Ishikawa, Masato and Asai, Kiyoshi and Nakai, Kenta'
title: Automatic extraction of motifs represented in the hidden Markov model from a number of DNA sequences
---
authors: 'Yadgari, J. and Amir, A.'
title: Genetic Threading
---
authors: 'Yan, Yan and Phan, Leuyen and Yang, Fan and Talpaz, Moshe and Yang, Yu and Xiong, Zeyu and Ng, Bernard and Timchenko, Nikolai A. and Wu, Catherine J. and Ritz, Jerome and Wang, Hong and Yang, Xiao-Feng'
title: A Novel Mechanism of Alternative Promoter and Splicing Regulates the Epitope Generation of Tumor Antigen CML66-L
---
authors: 'Yandell, Mark and Mungall, Chris J. and Smith, Chris and Prochnik, Simon and Kaminker, Joshua and Hartzell, George and Lewis, Suzanna and Rubin, Gerald M.'
doi: 10.1371/journal.pcbi.0020015
title: Large-Scale Trends in the Evolution of Gene Structures within 11 Animal Genomes
---
authors: 'Yang, Ziheng and Nielsen, Rasmus'
title: Estimating Synonymous and Nonsynonymous Substitution Rates Under Realistic Evolutionary Models
---
authors: 'Yang, George P. and Ross, Douglas T. and Kuang, Wayne W. and Brown, Patrick O. and Weigel, Ronald J.'
title: Combining SSH and cDNA microarrays for rapid identification of differentially expressed genes
